Classes/Service/DyncssService.php

```diff
@@ -43,13 +43,22 @@ public static function getCompiledFile($inputFile)
     protected static function fixPathForInput($file)
     {
         if (TYPO3_MODE === 'FE') {
```

Comment:
Maybe throwing error if $file is empty/null or at least return something to identify that $file must not be empty (or set a default value $file="" in the parameters)

Reply:
I think the `\empty` throws an error, the PR checker should be updated accordingly I think.